packages/create-svelte/tsconfig.json

```diff
@@ -4,8 +4,8 @@
 		"checkJs": true,
 		"noEmit": true,
 		"noImplicitAny": true,
-		"target": "es2020",
```

Comment:
this isn't necessarily a small change. everytime we've tweaked this it's had an impact on some platform and I wouldn't be surprised if it introduces some breakage somewhere. it'd probably be best to do in a separate PR with a changeset if it's necessary

Reply:
it doesn't matter anyway, this PR is arguably redundant since #3458 and the zillion follow-ups. going to close it

Reply:
Ooh. @antony's PR made the same change though, which I don't feel great about. Is necessary or do you think we can back that part out?